rero_ils/modules/vendors/jsonschemas/vendors/vendor-v0.0.1.json

```diff
@@ -44,19 +65,36 @@
       "title": "Communication language",
       "type": "string",
       "enum": [
-        "ger",
-        "eng",
         "fre",
-        "ita"
-      ]
+        "eng",
+        "ita",
+        "ger"
+      ],
+      "form": {
+        "options": [
+          {
+            "value": "fre",
+            "label": "French"
+          },
+          {
+            "value": "eng",
+            "label": "English"
+          },
+          {
+            "value": "ita",
+            "label": "Italian"
+          },
+          {
+            "value": "ger",
+            "label": "German"
+          }
+        ]
+      }
     },
     "default_contact": {
       "title": "Default contact details",
       "description": "",
       "type": "object",
-      "required": [
-        "city"
-      ],
```

Comment:
Why removing `city` from required field ?
It seems important to have some required informations if user would define a `defaullt_contact`; at least `contact_person`

Reply:
ok